starter-code/js/index.js

```diff
@@ -38,13 +49,17 @@ function createNewItem(){
 
 }
 
+
+
 window.onload = function(){
   var calculatePriceButton = document.getElementById('calc-prices-button');
   var createItemButton = document.getElementById('new-item-create');
   var deleteButtons = document.getElementsByClassName('btn-delete');
 
-  calculatePriceButton.onclick = getTotalPrice;
-  createItemButton.onclick = createNewItem;
+  calculatePriceButton.addEventListener("click", getTotalPrice);
```

Comment:
This was not needed in this case but it's a good step although it will be duplicated with your onclick